.github/workflows/prettier.yml

```diff
@@ -0,0 +1,13 @@
+name: Formatting
+on: [push]
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
+      - name: prettier
+        run: |
+          npx prettier "**/*.{js,html,md,yml,yaml,json}" --check
```

Comment:
Please exclude js for now. Even though it is not impacting this product at the moment, I want to onboard it alongside EDA. I believe we will move in that direction but I want this to be a deliberate and informed choice on the part of the team.

Reply:
@MatthewBlanski I just wanted this to open the discussion! Will make the change.